ui/src/layouts/Notebook.vue

```diff
@@ -64,6 +64,9 @@ export default {
                     }
                     return true
                 })
+                .sort((a, b) => {
```

Comment:
These orderedGroups computed methods look pretty similar. We could extract the logic in a composable/mixin so we don't repeat the code everywhere

Reply:
Will conduct as a follow on task - don't need it blocking this as is